packages/app/features/dashboard/profile/profileRow.tsx

```diff
@@ -7,17 +7,21 @@ type ProfileRowProps = {
   icon: ReactElement;
   title: string;
   number?: number;
+  onPress?: () => void;
 };
 
-export function ProfileRow({ icon, number, title }: ProfileRowProps) {
+export function ProfileRow({ icon, number, title, onPress }: ProfileRowProps) {
   return (
-    <Pressable className="flex flex-row justify-between py-1.5">
+    <Pressable
+      className="flex flex-row justify-between py-1.5"
+      onPress={onPress}
+    >
       <View className="flex flex-row items-center">
         {icon}
         <Text className="font-bold ml-2 text-sm">{title}</Text>
       </View>
       <View className="flex flex-row items-center">
-        <Text className="text-sm">{number}</Text>
+        <Text className="text-sm">{number && number > 0 ? number : ""}</Text>
```

Comment:
maybe we can rename `number` to `trailingText` with an empty string as a default value?